src/api/types.ts

```diff
@@ -7,24 +7,39 @@
  *
  * The Original Developer is the Initial Developer. The Initial Developer of the Original Code is Atypon Systems LLC.
  *
- * All portions of the code written by Atypon Systems LLC are Copyright (c) 2019 Atypon Systems LLC. All Rights Reserved.
+ * All portions of the code written by Atypon Systems LLC are Copyright (c) 2022 Atypon Systems LLC. All Rights Reserved.
  */
 
-import client from '../client'
+import { ManuscriptSnapshot } from '../lib/doc'
 
-interface InvitedUser {
-  email: string
-  name?: string
+export type TransformVersionResponse = {
+  transformVersion: string
 }
 
-export const projectInvite = (
-  projectID: string,
-  invitedUsers: InvitedUser[],
-  role: string,
-  message = 'message'
-) =>
-  client.post(`/invitation/${encodeURIComponent(projectID)}/invite`, {
-    invitedUsers,
-    role,
-    message,
-  })
+export type SendStepsPayload = {
+  steps: unknown[]
```

Comment:
Why steps type defined as unknown?

Reply:
This is how it was before. I updated it now to be `Steps[]`.